src/edn.rs

```diff
@@ -10,35 +10,26 @@ pub fn field_to_keyword(field_name: String) -> String {
 
 #[test]
 fn test_field_to_keyword_lowercase() {
-    assert_eq!(field_to_keyword("name".to_string()), ":name");
-    assert_eq!(field_to_keyword("crux__db___id".to_string()), ":crux.db/id");
-    assert_eq!(
-        field_to_keyword("account___amount".to_string()),
-        ":account/amount"
-    );
-    assert_eq!(field_to_keyword("tx___tx_time".to_string()), ":tx/tx-time");
+    assert_eq!(field_to_keyword("name"), ":name");
+    assert_eq!(field_to_keyword("crux__db___id"), ":crux.db/id");
+    assert_eq!(field_to_keyword("account___amount"), ":account/amount");
+    assert_eq!(field_to_keyword("tx___tx_time"), ":tx/tx-time");
 }
 
 #[test]
 fn test_field_to_keyword_mixedcase() {
-    assert_eq!(field_to_keyword("Name".to_string()), ":name");
-    assert_eq!(field_to_keyword("Crux__dB___id".to_string()), ":crux.db/id");
-    assert_eq!(
-        field_to_keyword("acCount___amouNt".to_string()),
-        ":account/amount"
-    );
-    assert_eq!(field_to_keyword("tX___tx_timE".to_string()), ":tx/tx-time");
+    assert_eq!(field_to_keyword("Name"), ":name");
+    assert_eq!(field_to_keyword("Crux__dB___id"), ":crux.db/id");
+    assert_eq!(field_to_keyword("acCount___amouNt"), ":account/amount");
```

Comment:
is the CAPS intentional? `acCount___amouNt`

Reply:
Yup

Reply:
Every test here checks that all characters will become lowercase.